catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -722,8 +726,16 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
 
     try {
       LOGGER.debug("WFS Source {}: Getting hits.", getId());
-      final WfsFeatureCollection hitsResponse = wfs.getFeature(getHits);
-      final long totalHits = hitsResponse.getNumberOfFeatures();
+
+      long totalHits;
+
+      try {
+        final WfsFeatureCollection hitsResponse = wfs.getFeature(getHits);
+        totalHits = hitsResponse.getNumberOfFeatures();
+      } catch (ResponseProcessingException e) {
+        LOGGER.info("got an error while querying for the hit count", e);
```

Comment:
✏️ nit but suggest re-wording this to something like 
`LOGGER.warn("WFS Source {} hit count query returned an unexpected exception", this.id, e)`

Reply:
I'll make the change